### DISCUSSION TITLE
Poor distinguishability/contrast in the dark mode contributions widget

### DISCUSSION THREAD

**Discussion by bl4ckswordsman** [community/community#147920] [Mobile]
There's very poor contrast in the contribution boxes of the dark mode widget in the GitHub for Android app, as seen below:
![Screenshot_20241225-191957.png](https://github.com/user-attachments/assets/6e1c5c00-c60d-4e11-b5ba-c40980d34b8c)

In the light mode of the widget, the contrast is much better:
![Screenshot_20241225-192043.png](https://github.com/user-attachments/assets/36eb6fab-942d-4b64-863a-ff0104ce6d97)

Even in the dark mode github.com mobile page, there's much better clarity in the contributions graph, compared to the dark mode widget:
![Screenshot_20241225-192404~2.png](https://github.com/user-attachments/assets/943aff8a-55e3-436d-bbc3-d0c8f8409699)

**Comment by aconsuegra** [comment]
Thanks for your feedback @bl4ckswordsman! I'll create an issue for our engineers so they can look into this.
